CSG/CSGImport.cc

```diff
@@ -309,7 +309,13 @@ CSGPrim* CSGImport::importPrim(int primIdx, const snode& node )
     sn::GetLVNodesComplete(nds, lvid);   // many nullptr in unbalanced deep complete binary trees
     int bn = nds.size();                 // binary nodes
 
-    // 2. count total subs for any listnodes of this lvid 
+    // Check if binary tree blows up and warn user
+    LOG_IF(warning, bn > 100000)
```

Comment:
The threshold of 100000 nodes seems a bit arbitrary. Should we make it configurable?